### DISCUSSION TITLE
Welcome to PopClip-Extensions Discussions!

### DISCUSSION THREAD

**Discussion by pilotmoon** [pilotmoon/PopClip-Extensions#1143] [General]
## 👋 Welcome!
  I've enabled the new discussions feature here on GitHub. Please use discussions to ask questions about extension development and interact with other users. Let's see how it goes. 
-Nick

**Comment by zaconline** [comment]
Does the **Formatting** **extension** work with Google docs?

**Reply by pilotmoon** [reply]
It does

**Comment by zaconline** [comment]
Is there a way to reorder the extensions that come up on the menu?  I would like to move the more recent ones I have added closer to the beginning on the menu.

**Reply by pilotmoon** [reply]
See https://www.popclip.app/guide/settings#re-ordering-actions

**Reply by pilotmoon** [reply]
![CleanShot 2024-02-23 at 09 53 12](https://github.com/pilotmoon/PopClip-Extensions/assets/17520/123f8c93-0149-43ee-a651-132f619e2040)

**Comment by extrastu** [comment]
I wrote an extension that I hope will be included, and I look forward to your reply.

https://github.com/extrastu/popclip-flomoplus

**Comment by zaconline** [comment]
Thank you.  I had tried that, but didn’t realize that you had to drag by the icon on the far left.

About the shortcut for bringing up pop clip when it disappears I know how to create a short cut when you are recording something that you can do, but in this case it is recording something you can’t do.  Hope that makes sense.  Popclips come up when you highlight something.  But making it come up when it disappears, how do you record that?  What am I missing?


Nina Zacuto
***@***.*** ***@***.***>
6221W. 5th St, Los Angeles, CA 90048
818-667-4840-c

**Reply by pilotmoon** [reply]
PopClip's keyboard shortcut causes PopClip to appear. I don't know how else to explain that! Have you tried it?
